ufo/ufo-profiler.c

```diff
@@ -95,6 +95,48 @@ ufo_profiler_new (void)
     return UFO_PROFILER (g_object_new (UFO_TYPE_PROFILER, NULL));
 }
 
+static void
+_ufo_profiler_call (UfoProfiler    *profiler,
```

Comment:
Is the Python style allowed?

Reply:
Well, it's not necessary because of `static` but so what, it's implementation detail.